src/ProjectOrigin.Vault/Database/Postgres/Scripts/v2/v2-0008.sql

```diff
@@ -0,0 +1,170 @@
+set client_min_messages = WARNING;
+DROP TABLE IF EXISTS qrtz_fired_triggers;
```

Comment:
Seems scary to delete a bunch of tables?
Wouldn't it be more save to check if table exists and only create if it didn't? So not deleting by acciedent if script gets renamed in the future. So this script is safe to be run 100 times without breaking / deleting data.